package.json

```diff
@@ -1,4 +1,14 @@
 {
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.18.2",
+    "nodemon": "^3.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "ts-loader": "^9.5.1",
+    "webpack": "^5.89.0",
+    "webpack-cli": "^5.1.4"
+  },
   "name": "chromacraze",
   "version": "0.0.0",
   "description": "A fast-paced multiplayer browser clicker game",
```

Comment:
- `main` should be updated to the location of our server's `index.js`, I believe.
- Can we add a "`start`" script here (i.e. `npm start`) that runs the express server, and confirm it's working?